src/setup-gcloud.ts

```diff
@@ -87,9 +69,19 @@ export async function run(): Promise<void> {
     if (exportCreds) {
       let credsPath = core.getInput('credentials_file_path');
       if (!credsPath) {
-        const runnerTempDir = process.env.RUNNER_TEMP;
+        // Note: We explicitly and intentionally export to GITHUB_WORKSPACE
+        // instead of RUNNER_TEMP, because RUNNER_TEMP is not shared with
+        // Docker-based actions on the filesystem. Exporting to GITHUB_WORKSPACE
+        // ensures that the exported credentials are automatically available to
+        // Docker-based actions without user modification.
+        //
+        // This has the unintended side-effect of leaking credentials over time,
+        // because GITHUB_WORKSPACE is not automatically cleaned up on
+        // self-hosted runners. To mitigate this issue, this action defines a
+        // post step to remove any created credentials.
+        const runnerTempDir = process.env.GITHUB_WORKSPACE;
```

Comment:
Should we name it `workspaceDir` since it's not the RUNNER_TEMP dir?

Reply:
Done